gita/__main__.py

```diff
@@ -363,6 +371,17 @@ def main(argv=None):
                     metavar='group-name',
                     required=True,
                     help="group name")
+    pg_remove = group_cmds.add_parser('remove', description='remove repo(s) from a group.')
```

Comment:
maybe call it something more specific, say `rmrepo`?

Reply:
yep, renamed.